src/sentry/utils/snuba.py

```diff
@@ -58,6 +58,122 @@ def timer(name, prefix='snuba.client'):
 )
 
 
+def transform_aliases_and_query(**kwargs):
+    """
+    Convert aliases in selected_columns, groupby, aggregation, conditions,
+    orderby and arrayjoin fields to their internal Snuba format and post the
+    query to Snuba. Convert back translated aliases before returning snuba
+    results.
+    """
+    sentry_snuba_map = {
```

Comment:
@dcramer Would like your feedback on these name mappings

Reply:
quickly skimmed -- generally looks like what id expect

Reply:
could we move this to a constant defined outside of this function? then i can also use it for search

Reply:
Is it possible for search to just call this same method instead of raw_query (for now at least)? AFAIK it should just work. I think it's better if endpoints are unaware of this mapping. As a follow up we should probably move this into the get_snuba_translators as hoff suggested, so the translation will all be done in one place and automatically on every snuba request.

Reply:
I'd rather not use the function for search as is because:
- there are no unit tests (probably should be added even if we're not going to use for search)
- search is simpler in that we don't need to worry about translating results back and also, search only needs to worry about translating conditions, not group by or filter keys
- search requires OR condition support

If you make this a standalone constant, I can replace this https://github.com/getsentry/sentry/blob/master/src/sentry/api/event_search.py#L40 so that they're consistent until we actually move this into `get_snuba_translators` or whatever we decide to do.

Reply:
Yeah we can do that. When we do add to get_snuba_translators search code will probably have to change quite a bit in that case though and probably won't have the option of opting out of the translation on groupby and filterkeys anyway. Will add unit tests for this function as well, but each case is tested via discover currently. We can fairly easily add OR now if that's the dealbreaker.

Reply:
the dealbreaker is that this is a temporary solution and i don't think it's worth updating search when extracting the constant so it can be shared is a much smaller change.